### PR TITLE
[Term Entry] Java ArrayList .clear()

### LINKED ISSUE #4335: [Term Entry] Java ArrayList .clear()
### Reason/inspiration (optional)

Please add the `.clear()` method as a new term entry for [ArrayList](https://www.codecademy.com/resources/docs/java/array-list)�. Provide the entry in a markdown file and place under the file path `content/java/concepts/array-list/terms/clear/clear.md `

The entry should have the following:
�	An introductory paragraph about what the method is and does.
�	A syntax section with a pseudo code block with an explanation of it.
�	An example showing the method in use.

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by elifmo9**
### Description

- Added the .clear method for Java ArrayList
- Closes #4335 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/java/concepts/array-list/terms/clear/clear.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  - 'Java'
+  - 'Data Structures'
+Tags:
+  - 'Method'
+  - 'ArrayList'
+  - 'Java'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/java-developer'
+---
```
Comment (mamtawardhani) [inline content/java/concepts/array-list/terms/clear/clear.md:14]:
```suggestion
---
Title: '.clear()'
Description: 'Removes all elements from an ArrayList.' 
Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Arrays'
  - 'Methods'
  - 'Java'
CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---
```

Comment (mamtawardhani) [reply content/java/concepts/array-list/terms/clear/clear.md:14]: Added the appropriate tags from the documents

File: content/java/concepts/array-list/terms/clear/clear.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+The **`.clear()`** method removes all elements from an [`ArrayList`](https://www.codecademy.com/resources/docs/java/array-list) in Java.
```
Comment (mamtawardhani) [inline content/java/concepts/array-list/terms/clear/clear.md:16]:
```suggestion
The **`.clear()`** method in Java removes all elements from the [`ArrayList`](https://www.codecademy.com/resources/docs/java/array-list)

```

Comment (mamtawardhani) [reply content/java/concepts/array-list/terms/clear/clear.md:16]: Reframed this due to plagiarism issues

File: content/java/concepts/array-list/terms/clear/clear.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  - 'Java'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/java-developer'
+---
+
+The **`.clear()`** method removes all elements from an [`ArrayList`](https://www.codecademy.com/resources/docs/java/array-list) in Java. 
+
+## Syntax
+All elements of an instance of `ArrayList` can be cleared by passing the `.clear()` method. After invoking this method, the list will be empty, but the `ArrayList` instance remains with its current capacity unchanged. It does not return anything but rather empties the list.
```
Comment (mamtawardhani) [inline content/java/concepts/array-list/terms/clear/clear.md:19]: This particular line can be added in the introductory paragraph itself. You can remove it from here and add it in the above paragraph.

File: content/java/concepts/array-list/terms/clear/clear.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+```pseudo
+arrayListInstance.clear();
```
Comment (mamtawardhani) [inline content/java/concepts/array-list/terms/clear/clear.md:22]:
```suggestion
myArrayList.clear();
```

File: content/java/concepts/array-list/terms/clear/clear.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        
+        System.out.println("Fruits: " + fruits);  // Output: Fruits: [Apple, Banana, Cherry]
+        
+        // Clear the ArrayList
+        fruits.clear();
+        
+        System.out.println("Fruits after clear(): " + fruits);  // Output: Fruits after clear(): []
+    }
+}
+```
```
Comment (mamtawardhani) [inline content/java/concepts/array-list/terms/clear/clear.md:45]: Below this example, could you please also add the output using the ```shell ``` block?

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @elifmo9! Thank you for contributing to the Codecademy Docs! The entry is nicely written! 😄 

I've suggested a few changes, could you please review and modify them at your earliest convenience? Feel free to discuss any concerns, thank you! 😃

**Comment by elifmo9** [comment]
Hi @mamtawardhani ! Thank you for your feedback. I will work on these now and let you know when I have completed them! :)

**Comment by elifmo9** [comment]
Hi @mamtawardhani I have pushed the suggested changes to my branch. Does this automatically sync to the pull request or will I need to do  anything additional for this?

**Review by mamtawardhani — APPROVED** [review]
Thank you for the contribution @elifmo9 😃 

The entry looks good for a second round of review 🚀

**Review by Sriparno08 — APPROVED** [review]
Looks good! 👍

**Pull request merged**